packages/design-system-docs/src/example.js

```diff
@@ -31,14 +31,16 @@ function setModalExamples() {
   const dialogHeading = document.getElementById('dialog-heading');
   const dialogWrap = document.getElementById('dialog-wrap');
 
-  document.querySelectorAll('.dialog-open-btn').forEach((btn) => {
+  // Convert NodeList into an array with the slice.call() method because
+  // querySelectorAll() returns a NodeList and on IE 11, foreach() only works on Array objects
+  Array.prototype.slice.call(document.querySelectorAll('.dialog-open-btn')).forEach((btn) => {
```

Comment:
You might be able to use `Array.from` here just for simplicity, but otherwise this works!

Reply:
`Array.from` does look much better, but looks like it's not supported on IE unless additional polyfill is added. Let's stay with `Array.prototype.slice.call` for now